www/authoring/custom-ckeditor-toolbar.php

```diff
@@ -76,10 +76,12 @@
             </ul>
         </div>
         <div class="overview">
-            <h2>Customizing the RTE toolbar</h2>
-            <p>Customize the RTE toolbar to rearrange or hide specific options.</p>
-            <p>This demo demonstrates how you can customize the RTE toolbar to hide specific options which should not be made available during authoring.</p>
-            <p>See an article on how to <a href="https://help.learnosity.com/hc/en-us/articles/360014562597-Customizing-the-RTE-toolbar-in-Question-Editor-API">customize the RTE toolbar in Question Editor API</a>.</p>
+            <h2>Customizing the Rich Text Editor toolbar</h2>
+            <p>You can customize the Rich Text Editor (RTE) toolbar to rearrange or hide specific options for authors. For example, showing fewer buttons for a cleaner interface.</p>
+            <p>By default, the toolbar shows 28 buttons, like this:  
+            <img src="../static/images/ckeditor-default-toolbar.png" height=60% width=60%></p>
```

Comment:
@TheRealEdDawson  Do we want to re-capture that one without "text left-align" to be in focus?

Reply:
Oh, the image? Does it seem blurry for you?

Reply:
I meant focus on a single button... Discussed in slack and we will go with what we have for now.

Reply:
The blue selection from the left-align button being clicked, I see. I think it's maybe not a showstopper at this stage, but let's ask the Minions for a design review here. They might agree with your point.